clientconn.go

```diff
@@ -412,6 +413,16 @@ func WithChannelzParentID(id int64) DialOption {
 	}
 }
 
+// WithDisableServiceConfig returns a DialOption that specifies whether resolver
+// should fetch service config. Note that if the disableServiceConfig is set to true,
+// but resolver doesn't honor the setting and still fetch the service config and
+// returns it, grpc will ignore the config.
+func WithDisableServiceConfig(a bool) DialOption {
```

Comment:
Are there good reasons to accept a bool here?  If not, I'd prefer to leave it out and make it always disable it (like how `WithBlock` always makes it a blocking operation).